test/test_lazy.py

```diff
@@ -274,6 +278,107 @@ def lazy_to_eager(u):
     assert is_close, f"{lhs} not <= {rhs}"
 
 
+def _get_poiseuille():
+    p_low = 1
+    p_hi = 1.001
```

Comment:
@MTCam I turned the pressure way down from where it was in the Poiseuille example, and that seemed to fix the differences I was seeing between lazy and eager. Guessing that having that value be so large in relation to everything else was amplifying the rounding error. Is setting it like this ok?

Reply:
I can't think of an immediate problem with this.  The pressure ratio I used was pretty much arbitrary.